src/middlewared/middlewared/api/v25_10_0/core.py

```diff
@@ -212,15 +216,20 @@ class CoreSetOptionsOptions(BaseModel, metaclass=ForUpdateMetaclass):
         extra="ignore",
     )
 
+    legacy_jobs: bool
     private_methods: bool
     py_exceptions: bool
 
 
 class CoreSetOptionsArgs(BaseModel):
-    options: CoreSetOptionsOptions
+    options: CoreOptions
+
 
+class CoreSetOptionsResult(BaseModel):
+    result: CoreOptions
 
-CoreSetOptionsResult = single_argument_result(None, "CoreSetOptionsResult")
+    def to_previous(cls, value):
+        return None
```

Comment:
Not sure what this is.  Comment?

Reply:
It's documented in NEP-046, section "API versioning". Issue pending to copy this to the middleware developer documentation.